chingu-coders/Voyage2-Turtles-02
# persistent clock settings

## Purpose
* Makes user-selected clock options persist across sessions

## Approach/Fixes
* Just makes sense

## Testing
* Checkout this PR
* Change the clock settings
* refresh tab and confirm that changes to settings persist
